MOCC/src/configurations/test_aneesh/resource_manager_ds_setup.py

```diff
@@ -0,0 +1,12 @@
+import PyTango
```

Comment:
make code flake8 compliant
put everything inside functions and then use if name == 'main'